scripts/board_script.js

```diff
@@ -265,23 +271,31 @@ function changeCardPosition(oldid, newid, newlist) {
   saveConfiguration();
 }
 
+//funktion som raderar kort från tavlan och sedan sparar ändringar genom att anropa saveConfiguration funktion
 function removeCard(id) {
   document.getElementById("c" + id).remove();
   var cardIndex = configuration.findIndex( ({ cardid }) => cardid === id);
   configuration.splice(cardIndex, 1);
   saveConfiguration();
 }
 
+//function som sparar användarens kort
 function saveConfiguration() {
   localStorage.setItem(user, JSON.stringify(configuration));
   localStorage.setItem("cards", JSON.stringify(cards));
 }
 
+//funktion som ger möjlighet för användaren att logga ut
 function logout() {
   sessionStorage.clear();
   window.open("index.html", "_self");
 }
 
+//funktion som ger möjlighet för användaren att rensa sina kort från tavlan
+function clearBoard(){
```

Comment:
Att lägga till dessa kodrader gör att Kanvan-tavlan tas bort och skapas igen utan de raderade objekten. Visuellt ser det bra ut.
```suggestion
   document.getElementById("board").innerHTML = "";
   makeLists();
```